planet/reporting.py

```diff
@@ -110,3 +110,48 @@ def update(self,
 
         if self.bar is not None:
             self.bar.refresh()
+
+
+class AssetStatusBar(ProgressBar):
+    """Bar reporter of asset status."""
+
+    def __init__(
+        self,
+        item_type,
+        item_id,
+        asset_type,
+        disable: bool = False,
+    ):
+        """Initialize the object.
+        """
+        self.item_type = item_type
+        self.item_id = item_id
+        self.asset_type = asset_type
+        self.status = ''
+        super().__init__(disable=disable)
+
+    def open_bar(self):
+        """Initialize and start the progress bar."""
+        self.bar = tqdm(
+            bar_format="{elapsed} - {desc} - {postfix[0]}: {postfix[1]}",
+            desc=self.desc,
+            postfix=["status", self.status],
+            disable=self.disable)
+
+    @property
+    def desc(self):
+        return f'{self.item_type} {self.item_id} {self.asset_type}'
```

Comment:
Is the bar supposed to read `time - item_type item_ID asset_type - status: <status>`? I currently seems to have some square brackets where `item_type` should be.

When I run it on my computer I get this:
```
❯ planet data asset-wait PSScene 20221003_002705_38_2461 basic_udm2                
00:00 - [] 20221003_002705_38_2461 basic_udm2 - status: active
```